h/search/core.py

```diff
@@ -157,5 +157,8 @@ def _parse_aggregation_results(self, aggregations):
     def _get_total_hits(response):
         total = response["hits"]["total"]
         if isinstance(total, int):
-            return total  # ES 6.x
-        return total["value"]  # ES 7.x
+            # ES 6.x
+            return total
+
+        # ES 7.x
+        return total["value"]  # pragma: nocover
```

Comment:
This is work to extend our support for ES to 7, but we've not made the jump yet. When we do this coverage should break again and we can remove the 6 path.